### ISSUE TITLE
Multiple domains ports clash

### ISSUE THREAD

**Issue by algesten** [realityforge/chef-glassfish#14]
I try to define multiple domains like this:

```json
        "glassfish": {
            "domains_dir": "/usr/local/glassfish/glassfish/domains",
            "domains": {
                "domain1": {
                    "config": {
                        "port": 8080,
                        "admin_port": 4848,
                        "username": "admin",
                        "password": "secret"
                    },
                    ...
                "domain2": {
                    "config": {
                        "port": 8081,
                        "admin_port": 4849,
                        "username": "admin",
                        "password": "secret"
                    },
```

However I can't get it to start. Running the start line from /etc/init/glassfish-xxx by hand I can see the problem is ports clashing – but none of the configured ports.

One single domain listens to:

    4848 Admin port
    8080 HTTP Instance port
    8181 HTTP_SSL port
    3700 IIOP port
    8686 JMX_ADMIN port

I didn't configure SSL, and got the SSL port anyways. The IIOP port and JMX_Admin I can find in the generated domain.xml, but doesn't seem configurable.

**Comment by TeroPihlaja** [comment]
One option might be to use the asadmin portbase option:
https://blogs.oracle.com/alexismp/entry/flexible_glassfish_domain_creation_using

**Comment by TeroPihlaja** [comment]
Looks like another option could be to use --domainproperties option and to specify the different ports to asadmin

**Comment by realityforge** [comment]
This has been addressed in #62 by @TeroPihlaja 

Closing this issue as complete
